fiftyone/core/utils.py

```diff
@@ -1068,14 +1069,31 @@ def progress_dt(pb):
     return progress_dt
 
 
-class BaseBatcher(abc.ABC):
+class Batcher(abc.ABC):
+    """Base class for iterating over the elements of an iterable in batches.
+
+    Args:
```

Comment:
Always document kwargs of important classes, especially ones linked to from important public methods